.github/workflows/ci.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         # different operating systems
         os: [ubuntu-18.04, macos-10.15, windows-2019]
-        features: [--features ""]
+        features: [-v] # GitHub requires us to have something in this flag
```

Comment:
Our windows build will no longer let us pass the `--features` flag with an empty features set. So I tried different ways to get around this, but couldn't come up with anything clean. So adding a trivial flag like `-v` is what I came up with.

Reply:
I wonder if we should file an issue on cargo about this?

Reply:
I just installed Rust and tried this on my Windows machine and it seems to work fine with `--features ""`. So I'd guess it's either something with the Windows Server version it uses, or with the `retry` action?